cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -286,6 +286,16 @@ class SubstraitToVeloxPlanConverter {
       return true;
     }
 
+    /// Set the existence of or-range and returns whether it can coexist with
+    /// existing conditions for this field.
+    bool setOrRange() {
```

Comment:
Sorry but I just notice 'setMultiRange' is for the 'or' condition, so we can just use it.

Reply:
`setMultiRange` seems to conflict with `setCertainRangeForFunction` of 1681L.